build/ci/.azure-devops-android-tests.yml

```diff
@@ -110,7 +110,6 @@ jobs:
       ArtifactType: Container
 
 - job: Android_Tests
-  displayName: 'Run Android UI Tests'
```

Comment:
Why is this removed?

Reply:
it is a test of what is displayed without.

Reply:
I think it will display as "Android_Tests" (the job name)